README.md

```
@@ -47,38 +47,189 @@ parser parse -f markdown https://example.com/article
+1. **Package Import**: Use root package instead of `/pkg/parser`
+2. **Context Required**: All methods now require `context.Context` first parameter
+3. **Functional Options**: Use `hermes.WithXxx()` options instead of struct fields
+4. **Error Types**: New `*hermes.ParseError` type with error codes
+5. **HTTP Client**: Client manages its own HTTP client, configurable via options
+6. **Content Types**: Set via `WithContentType()` option, affects parser extraction
+
+### Options Mapping
+
+| Old API | New API |
+|---------|---------|
+| `parser.ParserOptions{ContentType: "markdown"}` | `hermes.WithContentType("markdown")` |
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)